js/crm.buttons.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Function to change button text and disable one it is clicked
+ * @deprecated
+ * @param obj object - the button clicked
+ * @param formID string - the id of the form being submitted
+ * @param string procText - button text after user clicks it
+ * @return bool
+ */
+/* Changes button label on submit, and disables button after submit for newer browsers.
+ Puts up alert for older browsers. */
+function submitOnce(obj, formId, procText) {
```

Comment:
@mattwire just looking at this it doesn't look like your changing the text at all

Reply:
@seamuslee001 I feel like we should drop that.  Otherwise it creates more complexity - eg. which one gets the "processing" text, as it doesn't make sense to only disable one of the buttons, and you get the feedback that all buttons are disabled